docs/howto/distributed-tracing/proc_tracing-a-complex-service.adoc

```diff
@@ -41,14 +41,14 @@ Here, you can see that not only JAX-RS resource methods can be traced.
 This is a regular method in a CDI bean.
 Unlike JAX-RS resources, which are traced automatically, other methods need the explicit `@Traced` annotation.
 
-The most important part is this usage of MicroProfile OpenTracing API:
+The most important part is this usage of Eclipse MicroProfile OpenTracing API:
```

Comment:
This is part of the "Distributed Tracing" guide. Could we make sure the guide uses the full name somewhere at the beginning, and then leave the "Eclipse" word out? It reads worse now.